pocket_coffea/scripts/dataset/dataset_query.py

```diff
@@ -225,6 +226,86 @@ def do_query_results(self):
             )
         else:
             print("First [bold red]query (Q)[/] for a dataset")
+    
+    def generate_default_metadata(self, dataset):
+        year = self.extract_year_from_dataset_name(dataset)
+        isMC = self.is_mc_dataset(dataset)
+        xsec = self.extract_xsec_from_dataset_name(dataset)
+        primary_dataset,year_data,era_data = self.extract_era_from_dataset_name(dataset)
+        if isMC == True:
+            return {
+                "year": year,
+                "isMC": isMC,
+                "xsec": xsec
+            }
+        else:
+            return {
+                "year": year_data,
+                "isMC": isMC,
+                "primaryDataset": primary_dataset,
+                "era": era_data
+            }
+
+    def extract_year_from_dataset_name(self, dataset_name):
+        pattern = r'\/([^\/]+)NanoAOD'
+        match = re.search(pattern, dataset_name)
+    
+        if match.group(1) == 'Run2016':
```

Comment:
we should make it consistent with the naming in: https://github.com/PocketCoffea/PocketCoffea/blob/main/pocket_coffea/parameters/lumi.yaml#L29

Reply:
I have fixed this and also removed the unnecessary xsection dictionary for now.